_snippets/qstash-common-request.mdx

```diff
@@ -1,3 +1,10 @@
+
+<ParamField path="destination" type="string" required>
+  Destination can either be a URL Group name or id that you configured in the
+  Upstash console, or a valid url where the message gets sent to. Make sure the
+  url is prefixed with a valid protocol (`http://` or `https://`)
+</ParamField>
```

Comment:
see commit fc381e0893af460033682cc01fdc51055b2e5f85

Reply:
We should revert this part. I put a separate destination to schedule is that, in schedule you can't use the api as the destination, but you can in publish and enqueue. So we have `_snippets/qstash-common-request.mdx` common for schedule, publish, and enqueue; `_snippets/qstash-common-publish-request.mdx` common for publish and enqueue (which depends on `_snippets/qstash-common-request.mdx`) and have a separate destination in schedule

Reply:
fixed [36afd94](https://github.com/upstash/docs/pull/217/commits/36afd946bbab2fe8ec5ed739dba08de3c5dec238)